lib/openstates/api.rb

```diff
@@ -15,6 +15,11 @@ def bills(options = {})
         end
       end
 
+      if options[:id]
+        url << "#{options[:id]}/"
+        options.delete(:id)
+      end
```

Comment:
It does. But IMHO it's best to leave the nomenclature the same way the official api use. This way, when going through all the ```OpenStates::Bill.where```'s results it's clear which attribute you need to use. I had some difficulties with it and had to go to the official documentation because it wasn't clear which id was right.